prometheus/promhttp/http.go

```diff
@@ -381,6 +387,14 @@ type HandlerOpts struct {
 	ProcessStartTime time.Time
 }
 
+type OpenMetricsOptions struct {
+	Enable bool
+	// If true, EnableUnit adds the unit to the encoder options, ultimately allowing the
```

Comment:
```suggestion
	// EnableUnit adds the unit to the encoder options, ultimately allowing the
```

(this is to comply with "godoc" https://tip.golang.org/doc/comment)